File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        count = [0]*101
        for num in nums:
            count[num] += 1
        Sum = 0
        for i in range(len(count)):
            if count[i] == 1:
                Sum+=i
        return Sum

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        // int sum = 0;
        // int[] values = new int[101];
        // Arrays.fill(values,0);
        // for(int num: nums){
        //     values[num]++;
        // }
        // for(int i = 0; i<values.length;i++){
        //     if(values[i]==1) sum+=i;
        // }
        // return sum;
        int sum = 0;
        HashMap<Integer,Integer> map = new HashMap<>();
        for(int num:nums){
            if(map.containsKey(num)) map.put(num, map.get(num)+1);
            else map.put(num,1);
        }
        for(Map.Entry<Integer,Integer> entry : map.entrySet()){
            if(entry.getValue()==1){
                sum+= entry.getKey();
            }
        }
        return sum;
    }
}